schema/spacewalk/common/tables/suseContentFilter.sql

```diff
@@ -0,0 +1,36 @@
+--
+-- Copyright (c) 2018 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+-- Red Hat trademarks are not licensed under GPLv2. No permission is
+-- granted to use or replicate Red Hat trademarks that are incorporated
+-- in this software or its documentation.
+--
+
+CREATE TABLE suseContentFilter(
+    id       NUMBER NOT NULL
+                 CONSTRAINT suse_ct_filter_id_pk PRIMARY KEY,
+    org_id   NUMBER NOT NULL
+                 CONSTRAINT suse_ct_project_oid_fk
+                     REFERENCES web_customer(id)
+                     ON DELETE CASCADE,
+    type     VARCHAR2(16) NOT NULL,
+    name     VARCHAR2(128) NOT NULL,
+    criteria TEXT
```

Comment:
Forgot that oracle does not have "TEXT". It is called "CLOB" there.
I know that we do not (want to) support oracle anymore, but maybe we should keep it compatible as long as we did not drop it from code.